group-coordinator/src/main/java/org/apache/kafka/coordinator/group/assignor/GroupSpecImpl.java

```diff
@@ -56,11 +66,27 @@ public SubscriptionType subscriptionType() {
         return subscriptionType;
     }
 
+    /**
```

Comment:
nit: Should we use `{@inheritDoc}` instead of copying the javadoc? I would change it for all the methods coming from the interface.